modules/ROOT/pages/notifications/index.adoc

```diff
@@ -1,35 +1,203 @@
+Depending on the application, some of the fields from the notification object might not be visible.
 
-The following categories exist, ordered by severity:
+The notification object consists of the following fields:
 
+.Neo4j notification object
+[cols="<1s,<4"]
+|===
+|Neo4j code
+a|The Neo4j status code in the form of `Neo.ClientNotification.[SubType].[Name]`.
+|Title
+a|The description of the Neo4j status code.
+|Severity level
+a|The severity can be one of the following:
+
+- `WARNING`: There might be a problem with your query. Please, take a look.
+- `INFORMATION`: The query is correct, but this information can still be useful.
+|Category
+a|The category of the notification.
+|Position
+a|The position, given by row and column, where the notification is relevant in the query text.
+|===
+
+
+[role=label--version-5.23]
+[[gqlstatus-notification-object]]
+== GQL-status notification object
+
+The Neo4j GQL-status notifications are an implementation-defined subset of the GQL-status objects that cover informational notes and warnings, but not errors, `SUCCESS`, `NO DATA`, or `OMITTED RESULT`.
+For the latter, see <<general-codes-for-success, General codes for success>>.
+
+Each GQL-status object contains diagnostic information about the execution outcome.
+This information includes the GQLSTATUS code, the execution outcome (successful completion, warnings, no data, information), the status description, and the diagnostic record. +
+The main GQL-status object, describing the condition with the greatest precedence, is called the primary GQL-status object.
+The primary GQL-status object is always present in the execution outcome and follows the GQL precedence rules:
+
+. `NO DATA` has precedence over `WARNING`.
+. `WARNING` has precedence over the `SUCCESSFUL COMPLETION` subclass.
+. `SUCCESSFUL COMPLETION` subclass has precedence over `INFORMATIONAL`.
+. `INFORMATIONAL` is the condition with the least precedence.
+
+The primary GQL-status object also includes the Neo4j-specific information, such as the severity level and the classification of the notification, which can be used for filtering.
+For more information about notification grouping and filtering, see <<notification-grouping-and-filtering>>.
+
+The additional GQL-status objects are part of the list of additional GQL-status objects of the execution outcome.
+The list does not follow any specific order and might be empty as well.
+
+The GQL-status object consists of the following fields:
+
+.GQLSTATUS notification object
+[cols="<1s,<4"]
+|===
+|GQLSTATUS code
+a| A 5-character string that is the concatenation of a 2-character class code followed by a 3-character subclass code.
+|GQLSTATUS description
+a| A human-readable description of the GQLSTATUS, which consists of a condition, a subcondition, and a description.
+The condition and subcondition are the class and subclass codes, respectively.
```

Comment:
```suggestion
The condition and subcondition represents the class and subclass codes, respectively.
```

Not sure if represents is a good word. Possible you can write out that they are textual representations of the class and subclass codes or something. But the class and subclass codes the 5-digit alphanumeric ones while condition and subcondition is text